### PR TITLE
Added the switchToSeparateWindow method

### PR DESCRIPTION
**Pull request by gammadejs**

### CODE REVIEW

**Review by WenjiZ — DISMISSED** [review]
LGTM!

**Review by Jack-Vu — DISMISSED** [review]
LGTM!

**Review by ntnguyen328 — DISMISSED** [review]
Looks good!

**Review by KrizelC — DISMISSED** [review]

**Review by jolenejones — DISMISSED** [review]
LGTM! I think we did the same util <3333

File: src/test/java/utilities/Util.java [hunk 1]
```diff
@@ -15,4 +19,25 @@ public static void wait(int seconds) {
 		}
 	}
 
+	/*
+	 * Switches the driver focus to the child window
+	 * 
+	 * @Author: Erlyn Gammad
+	 */
+
+	public static void switchToSeparateWindow(WebDriver driver) {
```
Comment (boratech-macmini) [inline src/test/java/utilities/Util.java]: 1, What this function does, is basically `switchingToTheSecondAvailableWindow`
2, JavaDoc should be right above the function without empty line
3, Fix formatting

**Review by boratech-macmini — CHANGES_REQUESTED** [review]

**Review by boratech-macmini — APPROVED** [review]

**Pull request merged**